HIP/hip-1082.md

```diff
@@ -0,0 +1,225 @@
+---
+hip: 1082
+title: Allow Setting of EVM Address by EOA
+author: Nana Essilfie-Conduah <@Nana-EC>
+working-group: Richard Bair <@rbair>, Atul Mahamuni <@atul-hedera>, Ty Smith <ty-swirldslabs>,
+Serg Metelin <@sergmetelin>, Ali Nikan <@alinik4n>, Greg Scullard <@gregscullard>
```

Comment:
Should stay on one line so github doesn't complain about invalid yaml when rendering.

Reply:
Updated